src/stale_deps.jl

```diff
@@ -5,7 +5,13 @@ Test that `package` loads all dependencies listed in `Project.toml`.
 Note that this does not imply that `package` loads the dependencies
 directly, this can be achieved via transitivity as well.
 
-!!! note "Known bug"
+!!! note "Weak dependencies and extensions"
+
+    Due to the automatic loading of package extensions once all of
```

Comment:
For me it's not that they can't have any stale dependencies, it's that we cannot check for them with Aqua.

As a side note, the dependencies of an extension (the `using` statements at the top) are a superset of its triggers (that are necessary for it to load) because they also include the dependencies of the parent package itself